python/tvm/relay/frontend/pytorch.py

```diff
@@ -2260,7 +2260,8 @@ def broadcast_tensors(self, inputs, input_types):
         tensor_list = inputs[0]
         import torch
 
-        res_shape = list(torch.broadcast_shapes(*[self.infer_shape(t) for t in tensor_list]))
+        infer_shape_value = [self.infer_shape(t) for t in tensor_list]
```

Comment:
If we are trying to be backward-compatible here, would that be possible that we can check if the `broadcast_shape` available first, and fall back to `broadcast_tensors.shape` if not so?

Reply:
@shingjan Thanks for your suggestion. Adding an available check seems a better solution. This pr has been changed!